packages/dashboard/src/components/side-panel/shared/expandableSectionHeader.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { Icon } from '@cloudscape-design/components';
+import './styles.css';
+
+const ExpandableSectionHeader: FC<{ text: string; onClickButton?: (e: Event) => void }> = ({ text, onClickButton }) => {
+  return (
+    <>
```

Comment:
Shouldn't this have some semantic name? If it's a header can we take in a variant, but provide a default? If it's a header, `onClickButton` doesn't make sense as a prop name.

`type HeaderVariant = 'h1' | 'h2' | ... 'h6';`

Reply:
It is more like a header with an icon button that allows users to click the icon button for other functionality, like adding a new property to the properties setting section. So we do need onClickButton for that.